spec/features/user_can_sign_up_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+feature 'user can sign up' do
+  before do
+    visit root_path
+    click_on 'Sign Up'
+  end
+
+  it 'user should be on the sign up page' do
+      expect(page).to have_content 'Sign up'
+  end
+
```

Comment:
it would have been good to see some sad paths tested here

Reply:
sorry, i have done two now.